python/foglamp/common/configuration_manager.py

```diff
@@ -1361,6 +1361,8 @@ def _str_to_ipaddress(item_val):
                 return False
         elif _type == 'string':
             return isinstance(_value, str)
+        elif _type == "northTask":
+            return isinstance(_value, str)
```

Comment:
Seems like it is same as of **string** type. Though I am not sure about its  data validation w.r.t doc i.e `allow for items to contain the name of a north task configuration category`

Reply:
As per discussed with @MarkRiddoch `Correct, the main difference is that it is a hint to the GUI that this is a pick list of north task names. I don't think we can easily tell if the entry is that of a task, which would be a good check to be able to make. However it we can't easily do that then I would skip this for the time.`